src/components/settings/SettingsLMSTab/index.jsx

```diff
@@ -111,6 +119,8 @@ export default function SettingsLMSTab(enterpriseId) {
   useEffect(() => {
     // On load fetch potential existing configs
     fetchExistingConfigs();
+    // Check if the customer needs an idp config
+    setDisplayNeedsSSOAlert(enableSamlConfigurationScreen && !identityProvider);
```

Comment:
hmm.. I wonder how we should use this flag `enableSamlConfigurationScreen` in this section of the UI. looks like prior SAML screen is completely hidden when this flag is false. For purposes of LMS settings UI, does this flag even matter?

Reply:
I talked with product as well as ECS and because there are instances where the user/customer doesn't have/need an idp but still wants an LMS config, we only want to force them to make one if they have the screen enabled.

Reply:
cool! ty